### DISCUSSION TITLE
Unusual scanning problem - iOS

### DISCUSSION THREAD

**Discussion by Melynt3** [capawesome-team/capacitor-mlkit#109] [Q&A]
Hi, i have unusual problem there, somehow the camera doesn't scan the codes when I scan them from the top, I have no idea what it could be.

It only works like that on iOS devices

I normally get logs with the scanned raw value but nothing happens with it xd seriously, if I set the phone slightly to a vertical position, it scans immediately without any problem, what could be the problem?

Did anyone occured such a problem. 

Thanks in advance

**Comment by robingenz** [comment]
Please create a bug report with a reproduction example.

**Reply by Melynt3** [reply]
Firstly, i wanted to just ask if anyone had the same. 

If there will be none then ill just open it.
